src/nativeComponents/Button/Button.tsx

```diff
@@ -27,6 +27,11 @@ const styles = () => ( {
             opacity: 0.75,
         },
     },
+    actionButton: {
```

Comment:
let's keep the button more generic. Can't you pass this style from buttonStyle.root?